packages/ai-editor/src/browser/ai-editor-command-contribution.ts

```diff
@@ -175,3 +186,11 @@ export class AiEditorCommandContribution implements CommandContribution, MenuCon
         });
     }
 }
+
+function containsPrompt(args: unknown[]): args is (unknown | { prompt: string })[] {
```

Comment:
`Array<unknown | something>` doesn't narrow the type from `Array<unknown>`, since `unknown` itself includes all possible unions (except maybe `never`?). Is the type predicate important, or would it make sense to return just `boolean`?